index.js

```diff
@@ -7,7 +7,46 @@ const categories = require('./src/data/categories.json').categories;
 const app = express();
 const PORT = 8080;
 
```

Comment:
You should have included usage of underscore.js in this PR otherwise it'd be difficult for the reviewer to see which libraries you are using